mtps-get-task

```diff
@@ -129,13 +130,19 @@ send_query() {
     echo "$answer"
 }
 
+deduplicate() {
+    # Remove duplicated words
+    # from a string with whitespace separated words
+    echo "$1" | tr ' ' '\n' | sort | uniq | paste -sd " " -
```

Comment:
what does `paste` command ?

Reply:
` paste -sd " " -` takes multiple lines from standard input and merges them into a single line, with each original line separated by a space.